drivers/storage/portworx/component/telemetry.go

```diff
@@ -1237,6 +1251,14 @@ func GetCCMListeningPort(cluster *corev1.StorageCluster) int {
 	return startPort + 20
 }
 
+func GetPxCOListeningPort(cluster *corev1.StorageCluster) int {
+	startPort := pxutil.StartPort(cluster)
+	if startPort == pxutil.DefaultStartPort {
+		return pxDefaultCOListeningPort
+	}
+	return startPort + 21
```

Comment:
Is +21 the default port for PXCOListening Port? 
If so, can we make it a variable?

Reply:
Sure let me make that change...